helpers/freedom_helper.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const Axios = require('axios');
+const https = require('https');
+const config = require('config/config');
+const accounts = require('freedom-accounts-util');
+
+const { rejectUnauthorized } = config.AXIOS;
+const {
+    base_url,
+    client_id,
+    user_agent,
+    endpoints,
+    client_secret
+} = config.FREEDOM;
+
+const DASHBOARD_SCOPES = config.ACCOUNTS_API.scopes.DASHBOARD;
+let new_axios = Axios.create({
+    httpsAgent: new https.Agent({
+        rejectUnauthorized: rejectUnauthorized
+    })
+});
+
+
+module.exports = {
+    get_user_information,
+    get_oauth_access_token
+};
+
+async function get_oauth_access_token(params) {
+    params.client_id = client_id;
+    params.client_secret = client_secret;
+
+    const { access_token } = await accounts.generate_token(DASHBOARD_SCOPES.USER_READONLY);
+    const url = `${base_url}${endpoints.OAUTH_ACCESS_TOKEN}`;
+    const options = {
+        headers: {
+            'Content-Type': 'application/json',
+            'Authorization': `bearer ${access_token}`,
+            'User-Agent': user_agent
+        }
+    };
```

Comment:
@pandafox @wildtron should we consider implementing caching of access token for the validity time so that we do not need to generate new token frequently?

Reply:
That's already done by accounts.generate_token